bin/tldr

```diff
@@ -92,7 +93,11 @@ if (program.list) {
 } else if (program.render) {
   tldr.render(program.render);
 } else if (program.args.length >= 1) {
-  tldr.get(program.args, program);
+  if (program.markdown) {
```

Comment:
No need to specify an if condition here. We are already passing the `program` variable to `tldr.get` which contains the markdown option. Let us do the final calculation in the `renderContent` function.